IBPSA/Fluid/Sources/Examples/PropertySource_T.mo

```diff
@@ -0,0 +1,128 @@
+  Modelica.Blocks.Sources.Ramp ramp(
+    height=-2,
+    duration=1,
+    offset=1) "Ramp for mass flow rate"
+    annotation (Placement(transformation(extent={{-100,60},{-80,80}})));
+  IBPSA.Fluid.Sensors.TemperatureTwoPort senTemT(
+    redeclare package Medium = Medium,
+    m_flow_nominal=1,
+    tau=0) "Temperature sensor for when using temperature input"
+    annotation (Placement(transformation(extent={{20,30},{40,50}})));
+  IBPSA.Fluid.Sensors.TemperatureTwoPort senTemXi(
+    redeclare package Medium = Medium,
+    m_flow_nominal=1,
+    tau=0) "Temperature sensor for when using Xi input"
+    annotation (Placement(transformation(extent={{20,-10},{40,10}})));
+  IBPSA.Fluid.Sensors.TemperatureTwoPort senTemC(
+    redeclare package Medium = Medium,
+    m_flow_nominal=1,
+    tau=0) "Temperature sensor for when using C input"
+    annotation (Placement(transformation(extent={{20,-50},{40,-30}})));
+equation
+  connect(bouXi.ports[1], proSouXi.port_a)
+    annotation (Line(points={{-40,0},{-10,0}}, color={0,127,255}));
+  connect(bouC.ports[1], proSouC.port_a)
+    annotation (Line(points={{-40,-40},{-10,-40}}, color={0,127,255}));
+  connect(bouT.ports[1], proSouT.port_a)
+    annotation (Line(points={{-40,40},{-10,40}}, color={0,127,255}));
+  connect(Xi.y, proSouXi.Xi_in[1])
+    annotation (Line(points={{-19,20},{0,20},{0,12}}, color={0,0,127}));
+  connect(C.y, proSouC.C_in[1])
+    annotation (Line(points={{-19,-20},{4,-20},{4,-28}}, color={0,0,127}));
+  connect(bouT.m_flow_in, ramp.y) annotation (Line(points={{-62,48},{-72,48},{-72,
+          70},{-79,70}}, color={0,0,127}));
+  connect(ramp.y, bouXi.m_flow_in) annotation (Line(points={{-79,70},{-72,70},{-72,
+          8},{-62,8}}, color={0,0,127}));
+  connect(ramp.y, bouC.m_flow_in) annotation (Line(points={{-79,70},{-72,70},{-72,
+          -32},{-62,-32}}, color={0,0,127}));
+  connect(T.y, proSouT.T_in)
+    annotation (Line(points={{-19,60},{-4,60},{-4,52}}, color={0,0,127}));
+  connect(proSouC.port_b, senTemC.port_a)
+    annotation (Line(points={{10,-40},{20,-40}}, color={0,127,255}));
+  connect(senTemC.port_b, sin.ports[1]) annotation (Line(points={{40,-40},{60,
+          -40},{60,2.66667}}, color={0,127,255}));
+  connect(senTemXi.port_b, sin.ports[2]) annotation (Line(points={{40,0},{50,0},
+          {50,-2.22045e-16},{60,-2.22045e-16}}, color={0,127,255}));
+  connect(senTemT.port_b, sin.ports[3]) annotation (Line(points={{40,40},{60,40},
+          {60,-2.66667}}, color={0,127,255}));
+  connect(proSouT.port_b, senTemT.port_a)
+    annotation (Line(points={{10,40},{20,40}}, color={0,127,255}));
+  connect(proSouXi.port_b, senTemXi.port_a)
+    annotation (Line(points={{10,0},{20,0}}, color={0,127,255}));
+  annotation (Icon(coordinateSystem(preserveAspectRatio=false)), Diagram(
+        coordinateSystem(preserveAspectRatio=false)),
+    Documentation(info="<html>
+<p>
+Example model that illustrates the use of
+the <code>PropertySource</code> model.
```

Comment:
Use the full name with <a href=...  Otherwise this will not get change when renaming the model